package.json

```diff
@@ -3,20 +3,32 @@
   "version": "1.0.0",
   "description": "",
   "main": "bindings/node",
-  "scripts": {
-    "test": "echo \"Error: no test specified\" && exit 1"
+  "author": "Miles Glapa-Grossklag <miles@glapa-grossklag.com>",
+  "contributors": [
+    "amaanq (Amaan Qureshi) <amaanq12@gmail.com>"
+  ],
+  "license": "MIT",
+  "bugs": {
+    "url": "https://github.com/duskmoon314/tree-sitter-thrift/issues"
   },
-  "author": "",
-  "license": "ISC",
   "dependencies": {
     "nan": "^2.15.0"
   },
   "devDependencies": {
+    "eslint": "^8.33.0",
+    "eslint-config-google": "^0.14.0",
     "tree-sitter-cli": "^0.20.6"
   },
+  "repository": "https://github.com/glapa-grossklag/tree-sitter-elsa",
+  "scripts": {
+    "build": "tree-sitter generate && node-gyp build",
+    "test": "tree-sitter test && script/parse-examples",
+    "parse": "tree-sitter parse",
+    "test-windows": "tree-sitter test"
+  },
   "tree-sitter": [
     {
-      "scope": "source.lc",
+      "scope": "source.elsa",
```

Comment:
The file extension should remain `.lc`, because this is what the official Elsa project website uses.